ports/sonic.mania/Sonic Mania.sh

```diff
@@ -13,21 +13,62 @@ else
 fi
 
 source $controlfolder/control.txt
-
 get_controls
 
-$ESUDO chmod 666 /dev/tty1
+# Source Device Info
+source $controlfolder/device_info.txt
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
 
+# Set variables
 GAMEDIR="/$directory/ports/sonicmania"
 > "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
 
+# Set current virtual screen
+if [ "$CFW_NAME" == "muOS" ]; then
+  /opt/muos/extra/muxlog & CUR_TTY="/tmp/muxlog_info"
+elif [ "$CFW_NAME" == "TrimUI" ]; then
+  CUR_TTY="/dev/fd/1"
+else
+  CUR_TTY="/dev/tty0"
+fi
+
 cd $GAMEDIR
 
-export SDL_GAMECONTROLLERCONFIG="$sdl_controllerconfig"
-export LD_LIBRARY_PATH="$GAMEDIR/libs:$LD_LIBRARY_PATH"
+# Exports
+export LD_LIBRARY_PATH="$GAMEDIR/libs":$LD_LIBRARY_PATH
```

Comment:
Why `$LD_LIBRARY_PATH` is outside of the quotes ? It shouldn't be a problem but this is unusual to me ^^

Reply:
Just how I like to do it. If a variable isn't being added to I don't quote it.